spec/services/catalog/create_request_body_from_spec.rb

```diff
@@ -8,6 +8,12 @@
     let(:sanitize_service_instance) { instance_double(Catalog::OrderItemSanitizedParameters, :sanitized_parameters => {:a => 1}) }
     let(:local_tag_service_instance) { instance_double(Tags::CollectLocalOrderResources, :tag_resources => ["a"]) }
     let(:remote_tag_service_instance) { instance_double(Tags::Topology::RemoteInventory, :tag_resources => ["b"]) }
+    let(:service_offering_response) do
+      TopologicalInventoryApiClient::ServiceOffering.new(:extra => {"survey_enabled" => true}, :source_id => "333", :name => "test-platform-name")
+    end
+    let(:source_response) do
+      SourcesApiClient::Source.new
```

Comment:
Can't we just give this a `:name => "something"` and then test down on line 40 that `:platform => "something"`?